Mathlib/GroupTheory/Sylow.lean

```diff
@@ -740,8 +750,6 @@ theorem characteristic_of_normal {p : ℕ} [Fact p.Prime] [Finite (Sylow p G)] (
   congr
   simp [eq_iff_true_of_subsingleton]
 
-end Pointwise
```

Comment:
If you golf the proof of `characteristic_of_normal` above with `infer_instance` after `unique_of_normal`, then you shouldn't even need to have a `Pointwise` section at all, I suspect.

Reply:
The Pointwise section existed already, and I added `end Pointwise` to soothe a linter who had been born after the file were merged with an unclosed section.

Reply:
Here you are removing `end Pointwise`, not adding it.

Reply:
(But I am saying that the Pointwise section need not exist at all, if you golf the proof of `characteristic_of_normal`).

Reply:
Oh, I see what happened!